src/KafkaFlow.IntegrationTests/Core/Bootstrapper.cs

```diff
@@ -332,7 +333,8 @@ private static void SetupServices(HostBuilderContext context, IServiceCollection
                                     .DefaultTopic(GzipTopicName)
                                     .AddMiddlewares(
                                         middlewares => middlewares
-                                            .AddCompressor<GzipMessageCompressor>()))));
+                                           .AddCompressor<GzipMessageCompressor>())));
+                });
```

Comment:
undo this before merging

Reply:
Done. Please review https://github.com/Farfetch/kafkaflow/pull/451/commits/9d46d65611b192b9628a8fbfd6cd20ce871014e5

Reply:
Done. Please review https://github.com/Farfetch/kafkaflow/pull/451/commits/9d46d65611b192b9628a8fbfd6cd20ce871014e5

Reply:
Done. Please review https://github.com/Farfetch/kafkaflow/pull/451/commits/9d46d65611b192b9628a8fbfd6cd20ce871014e5